.gitignore

```diff
@@ -27,4 +27,4 @@ examples/.cache
 cranelift/isle/veri/veri_engine/test_output
 crates/explorer/node_modules
 tests/all/pulley_provenance_test.cwasm
-artifacts
+/artifacts
```

Comment:
This doesn't look right

Reply:
Oh this is actually intentional, although I forgot to mention it. We `./crates/test-programs/artifacts` in the repo which was otherwise being ignored when new changes were being added to it and I was editing things around there so figured I'd fix it up here as well. This was added originally because our c-api documentation generated this folder in the root of the repo